server/app.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+// const db = require('../database/index.js');
+const app = express();
+// const sequelize = require('../database/Models');
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use('/rooms/:id', express.static(__dirname + '/../client/dist'));
+app.use('/rooms/:id', express.static(__dirname + '/../public'));
+
+const controllers = require('./Controllers');
```

Comment:
Just out of curiosity... in your database, you have the getPhotosByRoomId. Then in the Controllers, you are calling the getPhotosByRoomId function inside getRoomPhotosByNumber with the id from the request. then on the server, you are calling getRoomPhotosByNumber. Why? Is it like a convention? Or does it make the code more structured?

Reply:
Yeah I should probably make the function names more clear.. but pretty much trying to separate concerns into a bit of an MVC model.. server/app.js handles my routes, controller file receives requests accesses the model functions, does any necessary computing, and responds to client.  and my model funcs live in database/index.js and are responsible for actually performing operations on db.  Not really a need to break out controllers with this small of an app, but as it grows it would make more sense.